pootle/core/search/__init__.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
```

Comment:
For the same reason as previous, this isn't just about TM.  TM just happens to be the current use case.